bin/rubydoop-runner

```diff
@@ -0,0 +1,9 @@
+# encoding: utf-8
+
+require 'rubydoop.jar'
+require 'rubydoop/job_runner'
+
+require 'puck'
+$: << "classpath:#{Puck::Jar::JAR_APP_HOME}/lib"
+
+Java::JavaLang::System.exit(Java::OrgApacheHadoopUtil::ToolRunner.run(Rubydoop::JobRunner.new, ARGV.to_java(:string)))
```

Comment:
I'm not sure yet if it's a good idea to move the `JobRunner` to Ruby, but that aside I think this line belongs in that class (as a class level `main` or `run` or `start` method).